### ISSUE TITLE
Feature request: Count of tests as well as number of ACs referenced

### ISSUE THREAD

**Issue by campbellssource** [vegaprotocol/approbation#65]
There are lot of tests that are written do not reference Acs but counting them add assurance about the coverage.

This script could count these

e.g. count the number of it blocks
`it(**)`
or something else that @AndyWhiteVega might be able to mention

it might be hard to categorise these right now, but there could be a row under the table that provides the count so I can see number of total ACs referenced and total number of tests.

**Comment by campbellssource** [comment]
There is something in the readme about special codes for additional tests here: https://github.com/vegaprotocol/approbation#how-to-reference-acceptance-criteria-in-a-spec

**Comment by edd** [comment]
Yeah - that... it turns out... has never actually been addressed in the code 😊 

But if using that `-additional-criteria` suffix would be acceptable to @AndyWhiteVega /etc then I think it's a good way to go. It lets us map tests that *aren't* addressing a specific acceptance criteria to the specification & category that it's covering, which I think makes sense.

It's definitely *possible* to list total tests in a repo, but:
- given this now covers core and frontend, which includes python, go, feature files, typescript & JS, I'd have to manually work out which lines indicate a new test in a bunch of different frameworks
- the *original* point of approbation was to map tests to ACs. I'm alright with the remit expanding, but I think including labelless tests means we can't relate them to an AC, or a spec, or even a category, and that feels too broad
- Don't wanna 😮‍💨
